src/udp_wrap.cc

```diff
@@ -213,6 +214,25 @@ X(SetMulticastLoopback, uv_udp_set_multicast_loop)
 
 #undef X
 
+void UDPWrap::SetMulticastInterface(const FunctionCallbackInfo<Value>& args) {
+  UDPWrap* wrap;
+  ASSIGN_OR_RETURN_UNWRAP(&wrap,
+                          args.Holder(),
+                          args.GetReturnValue().Set(UV_EBADF));
+
+  CHECK_EQ(args.Length(), 1);
+
+  node::Utf8Value iface(args.GetIsolate(), args[0]);
+
+  const char* iface_cstr = *iface;
+  if (args[0]->IsUndefined() || args[0]->IsNull()) {
```

Comment:
Agreed... since the js code requires that the `interfaceAddress` be specified, or, if we switch to requiring it to be a string, I would recommend simply switching these to `CHECK` assertions... e.g. 

```js
CHECK(!args[0]->IsUndefined());
CHECK(!args[0]->IsNull());
Utf8Value iface(args.GetIsolate(), args[0]);
int err = uv_udp_set_multicast_interface(&wrap->handle_, *iface);
```

Also note: it should be unnecessary to use the `node::` prefix on the Utf8Value decl